services/event/tests/event_test.go

```diff
@@ -878,6 +1041,4 @@ func TestRemoveEventFavorite(t *testing.T) {
 	if !reflect.DeepEqual(event_favorites, &expected_event_favorites) {
 		t.Errorf("Wrong tracker info. Expected %v, got %v", &expected_event_favorites, event_favorites)
 	}
-
-	CleanupTestDB(t)
 }
```

Comment:
Small thing, might be a bigger project to do in a different PR (if so, create an issue for this), it is at the same scale as reformatting everything, but if we could convert all these magic "testid1" to constant variables that would be amazing.

Reply:
Yeah, this is something that should be resolved in a separate PR. I can go ahead and make an issue for that.